web_widget_mpld3_chart/static/src/js/web_widget_mpld3_chart.esm.js

```diff
@@ -15,14 +15,12 @@ export default class Mpld3ChartJsonWidget extends Component {
             script.text = this.props.record.data[this.props.name].script;
             this.widget.el.append(script);
         });
-        onWillStart(() =>
-            loadBundle({
-                jsLibs: [
-                    "/web_widget_mpld3_chart/static/src/lib/d3/d3.v5.js",
-                    "/web_widget_mpld3_chart/static/src/lib/mpld3/mpld3.v0.5.10.js",
-                ],
-            })
-        );
+        onWillStart(async () => {
```

Comment:
A small suggestion. Why don't you create an specific asset and load directly as loadBundle? This way you may add extra libraries if needed.

WDYT?

Reply:
I changed loadBundle to loadJS funtion because the old method now only accepts an asset as a parameter. For simplicity, I have added the two lines of loadJS. But you are right, if we create an asset for these two files also works. Do you prefer to change it?

Reply:
I think I was not clear enough.

The idea is to create a new asset called mpld3 and add the libraries. This is done by adding the list on the assets key in the manifest. This way we can extend in the future with child libraries of mpld or adding specific css or whatever we want

Reply:
I've been trying to fix the error, but it seems that Odoo doesn't load the bundle correctly. I'm going back to the first version using `loadJS`.